rasa/cli/arguments/train.py

```diff
@@ -10,8 +10,11 @@
 )
 from rasa.shared.constants import DEFAULT_CONFIG_PATH, DEFAULT_DATA_PATH
 
+USE_LATEST_MODEL_FOR_FINE_TUNING = "use latest model for finetuning"
```

Comment:
How about assigning the value of `out` to the finetune argument right away to avoid having this constant spread around?